### PR TITLE
remove Modal.svelte file (no longer used) and some layout/css tweaks

### PR DESCRIPTION
**Pull request by jokull**

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -185,6 +189,10 @@
 						{#if movie?.showtimes} <Showtimes showtimes={movie?.showtimes} /> {/if}
 					</div>
 				</div>
+
+				<div
+					class="absolute inset-x-0 bottom-0 h-24 bg-gradient-to-t from-black z-40 pointer-events-none"
+				/>
```
Comment (multivac61) [inline src/routes/+page.svelte]: Is it possible that this code causes the modal to lose its round shape on the bottom?


https://user-images.githubusercontent.com/4436129/231550695-ce94c9b3-ed01-4671-887c-c830fb6c3b77.MP4

Comment (multivac61) [reply src/routes/+page.svelte]: IMG_3461

The above video doesn't really show this well. See the red circles ⭕

Comment (jokull) [reply src/routes/+page.svelte]: image

Comment (jokull) [reply src/routes/+page.svelte]: Are you seeing it in this build?

Comment (multivac61) [reply src/routes/+page.svelte]: This is the link I visited on my iPhone 14 Pro https://hvaderibio-b0ee6qdg5-multivac61.vercel.app/

The behavior is different on Firefox on macOS and iOS Safari 🤔

CleanShot 2023-04-12 at 19 06 04

IMG_E71224B737BE-1

Comment (multivac61) [reply src/routes/+page.svelte]: Its the same on https://hvaderibio-git-fork-jokull-css-tweaks-multivac61.vercel.app/

Comment (jokull) [reply src/routes/+page.svelte]: Not seeing it on iOS, Firefox macOS or Safari macOS. Weird.

Comment (multivac61) [reply src/routes/+page.svelte]: Super weird, I tried it on another iPhone 13 with the same overflowing effect.

4C850AB3-A5D5-4D8D-9FAD-BF8BF021D3E2_1_105_c

Comment (multivac61) [reply src/routes/+page.svelte]: I got some screenshots from Android users. The first two look like yours..

Screenshot_20230413-092810
Screenshot_20230413_092322_com android chrome

The third one is... well... 

Screenshot_20230413-092702_Samsung Internet

**Pull request merged**